src/components/auction/OrderPlacement/index.tsx

```diff
@@ -295,6 +295,17 @@ const OrderPlacement: React.FC<OrderPlacementProps> = (props) => {
     maxAmountInput && onUserSellAmountInput(maxAmountInput.toExact())
   }, [maxAmountInput, onUserSellAmountInput])
 
+  const balanceString = React.useMemo(() => {
+    return account
+      ? chainId !== chainIdFromWeb3
+        ? 'Switch network'
+        : `${biddingTokenBalance?.toSignificant(6) || '0'} ${getTokenDisplay(
```

Comment:
If chainId !== chainIdFromWeb3 holds, we should not even try to get the balance in order to avoid the web3 error messages

Reply:
But if you wanna do it in a separate PR later, that is fine as well

Reply:
This shouldn't fetch the balance using `chainId`, seems like every call composes with `useActiveWeb3React` so the single source of truth of which chainId continues to be the provider. 

Anyways if something extra must be changed when `chainId !== chainIdFromWeb3` I will do it in another PR.